server/data/outOfServiceBedClient.test.ts

```diff
@@ -105,7 +105,7 @@ describeCas1NamespaceClient('OutOfServiceBedClient', provider => {
   })
 
   describe('get', () => {
-    it('should get all outOfServiceBeds', async () => {
+    it('makes a request to the outOfServiceBeds endpoint with a page number', async () => {
       const outOfServiceBeds = outOfServiceBedFactory.buildList(2)
```

Comment:
V minor, feel free to ignore: I think it would be worthwhile declaring '2' as a const like `const pageNumber = 2` and using that throughout the test just to be a bit clearer.